TLM/TLM/Properties/AssemblyInfo.cs

```diff
@@ -35,4 +35,4 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.*")]
 
-[assembly:TypeForwardedTo(typeof(ExtVehicleType))]
\ No newline at end of file
+// [assembly:TypeForwardedTo(typeof(ExtVehicleType))]
```

Comment:
Is this intentional?

Reply:
This forwarded one deleted type to the new assembly.
This turned out to break save games.
Now the old type restored but the new type was granted a new different namespace, this forwarding is not necessary and i think it also produces a compile warning because the forwarded (formerly deleted) type exists again.